### PR TITLE
Draft graph activity

### PR DESCRIPTION
**Pull request by eymericboyer**
Added a first draft of the graph activity. Couldn't make it work. To be done next sprint.

### CODE REVIEW

**Comment by eymericboyer** [comment]
No test yet as no graph yet. I'll push the working graph+tests next week.

**Comment by 0xdieman** [comment]
Just fixed a conflict

**Comment by 0xdieman** [comment]
I think you can just put a description of the PR rather than your name and the sprint as a title

File: app/src/main/java/com/sdp/swiftwallet/domain/model/CurrencyAdapter.java [hunk 1]
```diff
@@ -59,6 +63,18 @@ public ViewHolder(@NonNull View itemView) {
...
+
+                @Override
+                public void onClick(View view) {
+
+                    Intent intent = new Intent(context, CryptoGraphActivity.class);
+                    int pos = getAbsoluteAdapterPosition();
+                    intent.putExtra("currency", currencyArrayList.get(pos));
+                    context.startActivity(intent);
+                }
+            });
```
Comment (0xdieman) [inline app/src/main/java/com/sdp/swiftwallet/domain/model/CurrencyAdapter.java:77]: This I guess you can do in a nicer way with a lambda

```suggestion
            itemView.setOnClickListener( ( view ) -> {
                 Intent intent = new Intent(context, CryptoGraphActivity.class);
                 int pos = getAbsoluteAdapterPosition();
                 intent.putExtra("currency", currencyArrayList.get(pos));
                 context.startActivity(intent);
            });
```

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -54,6 +54,17 @@
...
+    <string name="title_activity_crypto_graph">CryptoGraphActivity</string>
+  
+    <!-- Strings used for fragments for navigation -->
+    <string name="first_fragment_label">First Fragment</string>
+    <string name="second_fragment_label">Second Fragment</string>
+    <string name="next">Next</string>
+    <string name="previous">Previous</string>
+
+    <string name="hello_first_fragment">Hello first fragment</string>
+    <string name="hello_second_fragment">Hello second fragment. Arg: %1$s</string>
```
Comment (0xdieman) [inline app/src/main/res/values/strings.xml:67]: Are these used anywhere ?

**Review by 0xdieman — APPROVED** [review]

**Review by Zotchh — APPROVED** [review]
Pretty much the same as what @0xdieman commented already

**Pull request merged**